### PR TITLE
845776 - Update UG document for ScreenTip in WinUI Ribbon

### PR DESCRIPTION
**Pull request by YuvarajMasilamani**
Hi @SubashiniMahendran 

I have updated the UG document for ScreenTip in WinUI Ribbon. Please review and let me know in case of concerns.

Regards 
Yuvaraj

### CODE REVIEW

File: winui/Ribbon/RibbonScreenTip.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: ScreenTip | WinUI | Ribbon | Syncfusion
+description: Learn more about the Quick Access ToolBar  in the Syncfusion WinUI Ribbon (SfRibbon) control.
```
Comment (SubashiniMahendran) [inline winui/Ribbon/RibbonScreenTip.md:4]: Learn more about the ScreenTip in

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/Ribbon/RibbonScreenTip.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+</ribbon:RibbonButton>
+</ribbon:RibbonGroup>
+</ribbon:RibbonTab>
+</ribbon:SfRibbon.Tabs>
+</ribbon:SfRibbon>
+
+{% endhighlight %}
+{% endtabs %}
+
+![ScreenTip in Ribbon control](Ribbon-ScreenTip-images/winui-ribbon-screentip.png)
```
Comment (SubashiniMahendran) [inline winui/Ribbon/RibbonScreenTip.md:156]: @YuvarajMasilamani, also include topic regarding title and content template customization

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/Ribbon/RibbonScreenTip.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+---
+
+# ScreenTip in WinUI Ribbon Control
+
+`ScreenTip` is an enhanced ToolTip and it is used to show a popup window when the mouse cursor is hovered over `RibbonItem`. The popup is used to provide details of the `RibbonItem’s` function. The ScreenTip consist of two parts `Title` and `Content`. User can customize the ScreenTip title and content by `TitleTemplate` and `ContentTemplate` properties.
+
+## Adding ScreenTip for ribbon items
+
+ScreenTip can be added as follows.
+
```
Comment (SubashiniMahendran) [inline winui/Ribbon/RibbonScreenTip.md:17]: ScreenTip can be set to each Ribbon item by using its `ScreenTip` property. The below code shows how to define the ScreenTip for ribbon items in Ribbon control.

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/Ribbon/RibbonScreenTip.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+title: ScreenTip | WinUI | Ribbon | Syncfusion
+description: Learn more about the ScreenTip in the Syncfusion WinUI Ribbon (SfRibbon) control.
+platform: winui
+control: Ribbon
+documentation: ug
+---
+
+# ScreenTip in WinUI Ribbon Control
+
+`ScreenTip` is an enhanced ToolTip and it is used to show a popup window when the mouse cursor is hovered over `RibbonItem`. The popup is used to provide details of the `RibbonItem’s` function. The ScreenTip consist of two parts `Title` and `Content`. User can customize the ScreenTip title and content by `TitleTemplate` and `ContentTemplate` properties.
```
[the remark this answers is not in the document]
Comment (YuvarajMasilamani) [reply winui/Ribbon/RibbonScreenTip.md]: Updated

File: winui/Ribbon/RibbonScreenTip.md (hunk 3 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/RibbonScreenTip.md:22]: resolved

File: winui/Ribbon/RibbonScreenTip.md (hunk 2 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/RibbonScreenTip.md:62]: Added

File: winui/Ribbon/RibbonScreenTip.md (hunk 1 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/RibbonScreenTip.md]: Updated

File: winui/Ribbon/RibbonScreenTip.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+# ScreenTip in WinUI Ribbon Control
+
+`ScreenTip` is an enhanced ToolTip and it is used to show a popup window when the pointer hovers over a ribbon item. The pop-up provides helpful details about the function of the ribbon item.
+
+The ScreenTip consist of two parts 
+    
+* **Title:** A property of a type object that specifes the title of the ScreenTip.
+* **Content:** A property of a type object that specifes the content of the ScreenTip.
+
```
Comment (SubashiniMahendran) [inline winui/Ribbon/RibbonScreenTip.md:18]: The ScreenTip has two properties.

* **Title:** Specifies the title of the ScreenTip.
* **Content:** Specifies the content of the ScreenTip.

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/Ribbon/RibbonScreenTip.md (hunk 5 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/RibbonScreenTip.md:18]: Updated

**Review by SubashiniMahendran — APPROVED** [review]

**Pull request merged**